src/components/ItemFilters/utils.ts

```diff
@@ -0,0 +1,50 @@
+import type { selectedFilters as selectedFiltersType } from "../../types/filterTypes"
+
+const isRecapLocation = (loc: string) => {
```

Comment:
<img width="949" alt="Screenshot 2023-11-02 at 1 35 48 PM" src="https://github.com/NYPL/research-catalog/assets/6901105/355debb6-a62d-4d82-886b-86a2095f32ef">

This we may need optional chaining or some other type of undefined check or other error handling behavior here.

Reply:
i fixed this! it was actually a pretty big bug.